.github/workflows/release_please.yml

```diff
@@ -54,3 +54,38 @@ jobs:
           skip_existing: true
         env:
           CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+
+      - name: Get PR data
+        uses: actions/github-script@v7
+        id: get_pr_data
+        with:
+          script: |
+            return (
+              await github.rest.repos.listPullRequestsAssociatedWithCommit({
+                commit_sha: context.sha,
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+              })
+            ).data[0];
+
+      - name: Update PR labels
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const prNumber = ${{ steps.get_pr_data.outputs.result.number }};
```

Comment:
```suggestion
            const prNumber = ${{ steps.get_pr_data.outputs.result }};
```

Reply:
Ah just noticed you return the whole PR, dismiss this

Reply:
Does this have to use the `fromJSON` function if the output is string encoded?